chain-test/src/e2e/ContractTestClient.ts

```diff
@@ -90,18 +132,48 @@ interface TestClientParams {
   mockedChaincodeDir?: string;
 }
 
+/**
+ * Test client parameters specifically for REST API connections.
+ * Extends base parameters with required apiUrl.
+ */
 type TestClientParamsForApi = TestClientParams & { apiUrl: string };
```

Comment:
to be honest I think such comments might be too verbose. Maybe even the type is self-explanatory, and the full information is redundant. The information that might be more useful here is something like:

> REST API client requires `apiUrl` parameter to be provided